developer_manual/scripts/theme-bootstrap.sh

```diff
@@ -3,6 +3,19 @@
 # Invoke this script with one argument, the new theme's name.
 # Written by Dmitry Mayorov <dmitry@owncloud.com> & Matthew Setter <matthew@matthewsetter.com>
 # Copyright (c) ownCloud 2018.
+
+apt update
+
+apt install -y unzip wget
+
+wget https://github.com/owncloud/theme-example/archive/master.zip
+
+unzip master.zip
+
+mv theme-example-master theme-example
+
+mv theme-example /var/www/owncloud/apps/theme-example
```

Comment:
``/var/www/owncloud/apps/``

**Warning:**
Depending if you use only the ``apps/`` folder or a secondary folder for external apps (non core shipped apps) like ``apps-external`` your target path must be set accordingly. Please take care on that because it is muc more work to move things post upgrading the instance... !

**Proposal:**
I would note above the command list the following:
```
Please use the correct apps folder for moving the ``theme-example``. This can be for example ``apps/`` or 
if you have a secondary apps folder ``apps-external/``. For more details look into the apps management 
section of this documentation. The example below uses a secondary apps folder as target location.
```

``mv theme-example /var/www/owncloud/apps/theme-example`` -->
``mv theme-example /var/www/owncloud/apps-external/theme-example``

Reply:
Thanks for the further input @mmattel. I'm working on revising the script today.